pocs/focuser/birger.py

```diff
@@ -112,7 +130,6 @@ def move_to(self, position):
         Does not do any checking of the requested position but will warn if the lens reports hitting a stop.
         Returns the actual position moved to in lens encoder units.
         """
-        self.logger.debug('Moving focus to {} encoder units'.format(position))
```

Comment:
We've long talked about making `logger` level that's lower-level than `DEBUG`. This is the kind of thing that would be nice there :(